CONTRIBUTING.md

```diff
@@ -161,6 +161,63 @@ the docs will be built under `doc/_build/html`. Detail information about documen
 ## CI
 All PRs triger a CI job to run linting, type checking, tests, and build docs. The CI script is located [here](https://github.com/SeldonIO/alibi/blob/master/.github/workflows/ci.yml) and should be considered the source of truth for running the various development commands.
 
+## Optional Dependencies
+
+Alibi uses optional dependencies to allow users the choice to not install large or error-prone dependencies. These are 
```

Comment:
Nitpick: "error-prone dependencies" suggests alibi might be error-prone! Can we say "... to allow users to avoid installing large or challenging to install dependencies" or something along those lines?